.github/workflows/lint-format.yaml

```diff
@@ -49,4 +52,4 @@ jobs:
         run: pnpm install
 
       - name: Run Format
-        run: pnpm format:check
+        run: npm run format:check
```

Comment:
Is there specific reason for not using pnpm? If so, then do we need to add pnpm-lock to prettier ignore?

Reply:
It does not seem like we need pnpm since it seems like we're only using npm (we only have package-lock.json committed).

I have refactored it to use npm.

If we want to use pnpm, let me know and I'll commit a pnpm lock and make it fully use pnpm.

Reply:
Refactored to use pnpm